### PR TITLE
Use a configuration file for mocha ut

### PR DESCRIPTION
**Pull request by jerome-benoit**
Signed-off-by: Jérôme Benoit <jerome.benoit@sap.com>

## PR Checklist

- [x] Please add a description of your changes.
- [ ] We need your changes to come with unit tests in order to keep this project in quality and easy to maintain.
- [ ] Please add a link to the open issue or task that this pull request will resolve.
- [ ] Add a note to the changelog to indicate the change.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

If yes please indicate it and migration info into the CHANGELOG.md file.

---

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
     "benchmark": "npm run build && node -r source-map-support/register benchmarks/bench.js",
     "benchmark:debug": "npm run build && node -r source-map-support/register --inspect benchmarks/bench.js",
     "benchmark:prod": "npm run build:prod && node -r source-map-support/register benchmarks/bench.js",
-    "test": "npm run build && nyc mocha -r source-map-support/register --exit --timeout 20000 --parallel 'tests/**/*.test.js'",
-    "test:debug": "npm run build && mocha -r source-map-support/register --inspect --exit --timeout 20000 'tests/**/*.test.js'",
-    "test:prod": "npm run build:prod && nyc mocha -r source-map-support/register --exit --timeout 20000 'tests/**/*.test.js'",
+    "test": "npm run build && nyc mocha --parallel 'tests/**/*.test.js'",
```
Comment (pioardi) [inline package.json:12]: we cannot add the parallel also to the file ?

Comment (jerome-benoit) [reply package.json:12]: That will make the debug difficult and probably not supported by most debugger client.

Comment (pioardi) [reply package.json:12]: Got it

**Review by pioardi — APPROVED** [review]
Thanks for this improvement my dear friend

**Pull request merged**